ansible/host_vars/localhost

```diff
@@ -0,0 +1,16 @@
+---
+# FIXME(w-miller): set this more cleverly?
+base_path: "~/tenks/"
```

Comment:
What about playbook_dir as in https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html

Reply:
Thanks, that's just what I was after.